pkgs/top-level/all-packages.nix

```diff
@@ -28266,6 +28266,8 @@ with pkgs;
 
   mm-common = callPackage ../development/libraries/mm-common { };
 
+  MoniqueMonosynth = callPackage ../applications/audio/MoniqueMonosynth { };
```

Comment:
```suggestion
  monique-monosynth = callPackage ../applications/audio/monique-monosynth { };
```

Reply:
Why not keep it `MoniqueMonosynth` this allows us to do `nix run 'nixpkgs#MoniqueMonosynth'`. Is there a guideline on which package name to choose?